repl/repl.go

```diff
@@ -1214,6 +1236,7 @@ var builtin = [...]commandDesc{
 	{"strict-builtin-errors", []string{}, "toggle strict built-in error mode"},
 	{"dump", []string{"[path]"}, "dump raw data in storage"},
 	{"help", []string{"[topic]"}, "print this message"},
+	{"target", []string{"[mode]"}, "set the runtime to exercise (default rego)"},
```

Comment:
😍 wasm on the repl

Reply:
❓ Should we perhaps mention "one of rego (default), wasm" or something like that? Right now, I have no way to figure out easily what other options there are:

```
> help
...
                target [mode] : set the runtime to exercise (default rego) 
...
> target
code bad arguments: target <mode>: expects exactly one argument
> target reg
invalid target "reg":must be one of {rego,wasm}
```

Reply:
Updated the command description.